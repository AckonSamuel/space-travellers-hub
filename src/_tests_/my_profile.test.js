import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Myprofile from '../Components/my_profile/my_profile';

it('Test if my_profile renderers correctly', () => {
    const tree = renderer.create(
        <Provider store={store}>
            <Myprofile />
        </Provider>,
    );
    expect(tree).toMatchSnapshot();
})