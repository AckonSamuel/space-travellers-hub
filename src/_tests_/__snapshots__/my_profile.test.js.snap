// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if my_profile renderers correctly 1`] = `
<div
  className="myProfile"
>
  <section
    className="my-comp"
  >
    <h2>
      My Mission
    </h2>
    <ul
      className="reservedCont"
    />
  </section>
  <section
    className="my-comp"
  >
    <h2>
      My Rockets
    </h2>
    <ul
      className="reservedCont"
    />
  </section>
  <section
    className="my-comp"
  >
    <h2>
      My Dragons
    </h2>
    <ul
      className="reservedCont"
    />
  </section>
</div>
`;
